### PR TITLE
[[ Xcode7.0 ]] We can't build iOS 9.0 on release-7.1.0, so we skip it

### PR DESCRIPTION
**Pull request by livecodesebastien**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -136,6 +136,17 @@ config-ios-%:
 
 compile-ios-%:
 	$(XCODEBUILD) -project "build-ios-$*$(BUILD_SUBDIR)/$(BUILD_PROJECT).xcodeproj" -configuration $(BUILDTYPE)
+	
+# Dummy targets to prevent our build system from building iOS 9.0 simulator and Device
+config-ios-iphonesimulator9.0:
+	@echo "Skipping iOS simulator 9.0 (not yet supported)"
+compile-ios-iphonesimulator9.0:
+	@echo "Skipping iOS simulator 9.0 (no yet supported)"
```
Comment (peter-b) [inline Makefile]: "no" -> "not"!

**Comment by peter-b** [comment]
@livecode-vulcan review ok 22990d7

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 22990d7529002f6ca5108dbf71722a6ff83f8343

**Comment by livecode-vulcan** [comment]
😞 test failure 22990d7529002f6ca5108dbf71722a6ff83f8343
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/434)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/434)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/433)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/432)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/438)
- try-community-win-x86: [failure](https://vulcan.livecode.com/builders/try-community-win-x86/builds/429)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/440)

**Comment by livecodesebastien** [comment]
@livecode-vulcan retry 22990d7

**Comment by livecode-vulcan** [comment]
😎 test success 22990d7529002f6ca5108dbf71722a6ff83f8343
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/435)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/435)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/434)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/433)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/439)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/430)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/441)

**Pull request merged**